tokio-console/src/view/help.rs

```diff
@@ -0,0 +1,70 @@
+use tui::{
+    layout::{self, Constraint, Direction, Layout},
+    text::Text,
+    widgets::{Clear, Paragraph, Wrap},
+};
+
+use crate::{state::State, view};
+
+/// Simple view for help popup
+pub(crate) struct HelpView<T> {
+    help_text: Option<T>,
+}
+
+impl<T> HelpView<T>
+where
+    T: Into<Text<'static>>,
+{
+    pub(super) fn new(help_text: T) -> Self {
+        HelpView {
+            help_text: Some(help_text),
+        }
+    }
+
+    pub(crate) fn render<B: tui::backend::Backend>(
+        &mut self,
+        styles: &view::Styles,
+        frame: &mut tui::terminal::Frame<B>,
+        _area: layout::Rect,
+        _state: &mut State,
+    ) {
+        let r = frame.size();
+        let content = self
+            .help_text
+            .take()
+            .expect("help_text should be initialized")
+            .into();
```

Comment:
it's kind of weird to me that `help_text` is an `Option` that we have to `take` here? Is there a reason the `render` method has to take an `&mut self`? It could just take `self` --- this isn't a trait method or anything.